### PR TITLE
Add a `tidequeryresults` metric for alerting on Tide query errors.

### PR DESCRIPTION
**Pull request by cjwagner**
/assign @listx

### CODE REVIEW

File: prow/tide/tide.go [hunk 1]
```diff
@@ -457,9 +468,15 @@ func (c *Controller) query() (map[string]PullRequest, error) {
 			go func() {
 				defer wg.Done()
 				results, err := search(c.ghc.QueryWithGitHubAppsSupport, c.logger, q, time.Time{}, time.Now(), org)
+
+				resultString := "success"
+				if err != nil {
+					resultString = "error"
+				}
+				tideMetrics.queryResults.WithLabelValues(strconv.Itoa(i), org, resultString).Inc()
```
Comment (listx) [inline prow/tide/tide.go:476]: Is the use of `i` here guilty of the same error described in this article https://dev.to/kkentzo/the-golang-for-loop-gotcha-1n35 ?

**Comment by petr-muller** [comment]
/cc @jmguzik 
FYI

File: prow/tide/tide.go (hunk 1 above)
Comment (cjwagner) [reply prow/tide/tide.go:476]: Nice catch, there was an existing problem with this 5 lines below as well. Should be fixed now.

**Pull request merged**